server/middleware/setUpSentry.ts

```diff
@@ -1,15 +1,30 @@
-import express from 'express'
 import * as Sentry from '@sentry/node'
-
-import config from '../config'
+import '@sentry/tracing'
+import express from 'express'
 import applicationVersion from '../applicationVersion'
+import config from '../config'
 
 export function setUpSentryRequestHandler(app: express.Express): void {
   if (config.sentry.dsn) {
     Sentry.init({
       dsn: config.sentry.dsn,
       environment: config.environment,
       release: applicationVersion.gitRef,
+      tracesSampler: samplingContext => {
+        const transactionName =
+          samplingContext && samplingContext.transactionContext && samplingContext.transactionContext.name
```

Comment:
I think this can be `const transactionName = samplingContext?.transactionContext?.name`